confidant/routes/credentials.py

```diff
@@ -6,6 +6,7 @@
 
 from datetime import datetime
 from flask import blueprints, jsonify, request
+from flask_restful import inputs
```

Comment:
Please don't add flask_restful, it's deprecated, and we're using blueprints for everything.